src/control/lib/control/pool_property_test.go

```diff
@@ -162,6 +162,17 @@ func TestControl_PoolProperties(t *testing.T) {
 			value:  "wat",
 			expErr: errors.New("invalid"),
 		},
+		"policy-valid": {
+			name:    "policy",
+			value:   "type=io_size",
```

Comment:
I am confused about how this is supposed to work. Looking at the server-side code, it seems as if the property is defined as something like `type=io_size,1,2`, where the numeric values specify the per-tier io size to be used as a threshold. But the properties parser won't allow this, as the delimiter between properties is `,`.

Reply:
Changed delimiter to ';' (semicolon)

Reply:
@kmajzero: I recommend that you merge my PR #6265 for the delimiter fix as well as latest master. The testing for my PR had many fewer failures. We're not going to be able to look at this PR until it's in better shape. Thanks.